lib/ui/viewer/search/collection_suggestions.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:photos/db/files_db.dart';
+import 'package:photos/ente_theme_data.dart';
+import 'package:photos/models/collection.dart';
+import 'package:photos/models/collection_items.dart';
+import 'package:photos/models/file.dart';
+import 'package:photos/services/collections_service.dart';
+import 'package:photos/ui/viewer/file/thumbnail_widget.dart';
+import 'package:photos/ui/viewer/gallery/collection_page.dart';
+import 'package:photos/utils/navigation_util.dart';
+
+class CollectionSuggestions {
+  final List<Collection> matchedCollections;
+  final BuildContext context;
+  const CollectionSuggestions(this.matchedCollections, this.context);
+
+  List<Widget> getSuggestions() {
+    List<Widget> collectionSuggestionWidgets = [];
+    collectionSuggestionWidgets =
+        generateSuggestionWidgets(collectionSuggestionWidgets);
```

Comment:
Do we need to pass `collectionSuggestionWidgets` as parameter in `generateSuggestionWidgets`? If not, may be we can declare `    List<Widget> collectionSuggestionWidgets = [];` inside that method and return it here,

Reply:
Done